### ISSUE TITLE
[BUG] Extended Info Not Working

### ISSUE THREAD

**Issue by Goldenfreddy0703** [SerpentDrago/skin.auramod#73]
Ok, this has never been mentioned as an issue and i really thought someone would make an issue for this. On Matrix, the addon script Extended Info is not working. I don't know where phil65 or scott967 with that script but i feel that extended info has been depreciated so as OpenMeta and OpenInfo sadly cause of TheMovieDb Helper. If someone could make pull request on these two addon scripts, i think we could get the extended info working again on Kodi Matrix. 

These two scripts are Matrix compatible but they need (Waiting on 3rd party addon fix)
https://github.com/scott967/script.extendedinfo/tree/Matrix
https://github.com/scott967/script.module.kodi65/tree/Matrix

If we could get this working, that would be the best. 
![Exteneded Info](https://user-images.githubusercontent.com/62456796/117541999-a6285f00-afe4-11eb-9de7-5874dcf362ba.PNG)

**Comment by SerpentDrago** [comment]
Honestly extended info is dead , dead dead dead , and will prob never be brought back . TMDB helper now offers everything a skin needs to properly implement its own Extended info like features (browsing cast , movies with that cast member and similar movies , etc etc , then being able to play them and pass off to an addon to find sources etc etc ) 

even auramod itself has all this in its "cast and more " now its currently broken and gets you stuck in a loop but its ACTIVELY being worked on and trying to be fixed . and its the #1 priority right now . 

The ability of Auramod to launch extended info from Netflix info button may even be removed. it only even existed in the first place when openinfo was active . it can i guess be kept and default'd to off just in case later a user does want that function they can enable the button , but its super low priority , and imho not needed anymore . 

Cast & more can even be expanded to provide more info if needed . once the skins major bugs are fixed

Skins that support this kinda browsing around and playback though tmdb helper are Auramod and Arctic Horizon . Fully working already in Arctic Horizon . Any skin could implement this functions and its best done on a SKIN level for various programing reasons and kodi limitations on addons vs skins (font support etc ) 

In the future if and when there is a extended info script to launch thats working if that button no longer works it could be looked into i guess . but we will devote no time in reviving a very very dead addon thats features have been replaced by native skin features . After all only skins can properly manage custom xmls with custom fonts and stuff . 

I'm closing this sorry , we are not interested . Have a great day !

**Comment by Goldenfreddy0703** [comment]
Aww ok thank you very much. I thought this was a big issue that has been going on. To remove the extended info button from Auramod, is there a setting for that or will there be a fix to get it removed for it in the future?

**Comment by SerpentDrago** [comment]
The button by default should be disabled I forget . but yes there is a setting to show or not show it . 

Skin Settings > library views > configure netflix video Info > Configure buttons > Extended info 

![image](https://user-images.githubusercontent.com/21133858/117545337-556c3280-aff3-11eb-96a0-d5dcf801c106.png)

**Comment by Goldenfreddy0703** [comment]
Thank you very much

**Comment by Goldenfreddy0703** [comment]
Hey so good news, Extended Info has been made and modded thanks to thenewdiamond repo from henryjfry. 
https://github.com/henryjfry/repository.thenewdiamond

This makes it so that Extended Info will work with Auramod.
